docs/src/administration/cli/api-tokens.md

```diff
@@ -153,19 +167,24 @@ To run a cron only on your production environment, check the environment type as
 
 ```yaml
 crons:
-    backup:
-        spec: 'H 4 * * *'
+    sendemails:
+        spec: '*/7 * * * *'
         commands:
             start: |
                 if [ "$PLATFORM_ENVIRONMENT_TYPE" = production ]; then
-                   platform backup:create --yes --no-wait
+                   cd public && send-pending-emails.sh
                 fi
 ```
 
-{{< note >}}
+## Use the CLI SSH certificate for other commands
+
+After you authenticate the CLI using a `PLATFORMSH_CLI_TOKEN` environment variable,
+you might want to run commands that aren't CLI commands.
+For example, you might want to run `ssh`, `git`, `rsync`, or `scp` commands.
 
-If you're running CLI commands from any automated system, including a Platform.sh cron task,
-it's best to use the `--no-wait` flag on any commands that may take more than a second or two to avoid blocking the process.
-Failure to do so may result in long deploy times and site downtime.
+To do so, load the CLI SSH certificate first by running the following command:
+
+```bash
+$ platform ssh-cert:load
```

Comment:
:hammer_and_wrench: I noticed this suggestion in the request for this change and it makes sense in most contexts where it happens (CI and other such non-interactive sessions).
```suggestion
$ platform ssh-cert:load --no-interaction
```